client/src/sockets/helpers/index.js

```diff
@@ -1,16 +1,37 @@
-import { ListEvents } from 'sockets/enums';
+import { CohortEvents, ListEvents } from 'sockets/enums';
 import history from 'common/utils/history';
+import { Routes } from 'common/constants/enums';
```

Comment:
I think it would be nice to extract helper functions as well from there so they can be used without needing to import both Routes and the routeGenerator. For example, `cohortRoute = id => routeGenerator(Routes.COHORT, id)`.